### ISSUE TITLE
child.ancestor.id in menu_tags is empty

### ISSUE THREAD

**Issue by TLuesebrinck** [django-cms/django-cms#8793]
Django-cms 5.1.1
I use the standard menu and dropdown bootstrap5 html files (from frontend). The aria-labelledby is invalid becuase the ancestor id is missing, every labelledby contians just "menu-"

```
  <div class="dropdown-menu" aria-labelledby="menu-{{ child.ancestor.id|safe }}">
      {% show_menu from_level to_level extra_inactive extra_active "bootstrap5/dropdown.html" "" "" child %}
  </div>
```
